docs/usage/templates.md

```diff
@@ -36,3 +36,23 @@ If you want to print pretty JSON with Handlebars you can use the built-in functi
 `{{{stringToPrettyJSON myvar}}}`
 
 In the example above `myvar` is a variable/field, that contains valid JSON.
+
+### containsString
+
+Returns `true` if a given string is a substring.
+
+`{{#if (containsString depName 'python')}}Python{{else}}Other{{/if}}`
+
+### and
+
+Returns `true` only if all expressions are `true`.
+
+`{{#if (and true true true)}}True{{else}}False{{/if}}` => `True`
+`{{#if (and false false)}}True{{else}}False{{/if}}` => `False`
+
+### or
+
+Returns `true` if at least one expression is `true`.
+
+`{{#if (or true true false)}}True{{else}}False{{/if}}` => `True`
+`{{#if (or false false)}}True{{else}}False{{/if}}` => `False`
```

Comment:
Please use a real example

Reply:
I would also like to see better formatting of the lines, right now it will wrap in bad places on the final output.

Would a table like format work better?

I was thinking of something like this, but that would only work if the input is not too big, or else we get the same wrapping problem.

| Input | Boolean output |
| --- | --- |
| `{{#if (or true true false)}}True{{else}}False{{/if}}` | True |
| `{{#if (or false false)}}True{{else}}False{{/if}}` | False |

Reply:
Is it ok then if I use the same variables from the test case?

Reply:
Updated.